src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/AddClientCertRunner.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.Commands
+{
+    public class AddClientCertRunner
+    {
+        public static void Run(AddClientCertArgs args, Func<ILogger> getLogger)
+        {
+            args.Validate();
+            var settings = RunnerHelper.GetSettings(args.Configfile);
```

Comment:
Maybe you can move the RunnerHelper static in it's own file. 
Now that it's used in multiple locations

Reply:
I am trying to touch as less as possible of your existing files. Like @zivkan said PR is already huge and it will not help to finally merge it. So I guess you can move this file in official repo :)

Reply:
Yep, makes sense :)